bind/motor-mobile/bind.go

```diff
@@ -214,22 +220,55 @@ func IssuePayment(buf []byte) ([]byte, error) {
 // Stat returns general information about the Motor node its wallet and accompanying Account.
 func Stat() ([]byte, error) {
 	if instance == nil {
-		return nil, errWalletNotExists
+		return nil, ct.ErrMotorWallet
 	}
 
 	doc := instance.GetDIDDocument()
 	if doc == nil {
-		return nil, errWalletNotExists
+		return nil, ct.ErrMotorWallet
 	}
-	diddoc, err := types.NewDIDDocumentFromPkg(doc)
+	didDoc, err := rt.NewDIDDocumentFromPkg(doc)
 	if err != nil {
 		return nil, err
 	}
 
 	resp := mt.StatResponse{
 		Address:     instance.GetAddress(),
 		Balance:     int32(instance.GetBalance()),
-		DidDocument: diddoc,
+		DidDocument: didDoc,
 	}
 	return resp.Marshal()
 }
+
+func BuyAlias(buf []byte) ([]byte, error) {
```

Comment:
`Buy Alias` should be implemented within the motor pkg and called from this wrapper. reduces duplication for given targets and make adding support for speedway easier.

Reply:
Shouldn't provide focus on only speedway. Should be holistic for all platforms, and keep in mind embeded targets